sources/packages/backend/libs/services/src/constants/error-code.constants.ts

```diff
@@ -32,3 +32,4 @@ export const UNEXPECTED_ERROR_DOWNLOADING_FILE =
 export const ECE_DISBURSEMENT_DATA_NOT_VALID =
   "ECE_DISBURSEMENT_DATA_NOT_VALID";
 export const FILE_PARSING_ERROR = "FILE_PARSING_ERROR";
+export const DOCUMENT_NUMBER_NOT_FOUND = "DOCUMENT_NUMBER_NOT_FOUND";
```

Comment:
Is this not more close to the integrations lib? Is there a reason to be created into the services lib?

Reply:
Reason being we don't have error-code.constants at integration level. I can create one.

Reply:
With the provided explanation there was no need to.